wenet/cli/paraformer_model.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import torch
+import torchaudio
+import torchaudio.compliance.kaldi as kaldi
+
+from wenet.paraformer.search import paraformer_greedy_search
+from wenet.utils.file_utils import read_symbol_table
+
+
+class Paraformer:
+
+    def __init__(self, model_dir: str) -> None:
+
+        model_path = os.path.join(model_dir, 'final.zip')
+        units_path = os.path.join(model_dir, 'units.txt')
+        self.model = torch.jit.load(model_path)
+        symbol_table = read_symbol_table(units_path)
+        self.char_dict = {v: k for k, v in symbol_table.items()}
+        self.eos = 2
+
+    def transcribe(self, audio_file: str, tokens_info: bool = False) -> dict:
+        waveform, sample_rate = torchaudio.load(audio_file, normalize=False)
+        waveform = waveform.to(torch.float)
+        feats = kaldi.fbank(waveform,
```

Comment:
The default window in the FunASR frontend is `hamming`. You can find more details [here](https://github.com/modelscope/FunASR/blob/42479a68cb3b2a95338d6c27a3ac9a5e66c60405/funasr/frontends/wav_frontend.py#L134). However, the default window in `kaldi.fbank` is `povey`, as specified [here](https://pytorch.org/audio/stable/generated/torchaudio.compliance.kaldi.fbank.html#torchaudio.compliance.kaldi.fbank). This different window maybe a little mismatch. As mentioned in line 44 of [this document](https://kaldi-asr.org/doc/feature-window_8h_source.html):
> "povey" is a window I made to be similar to Hamming but to go to zero at the edges, it's pow((0.5 - 0.5cos(n/N2*pi)), 0.85) I just don't think the Hamming window makes sense as a windowing function.